tappsbt/mock.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/lightninglabs/taproot-assets/internal/test"
 	"github.com/lightningnetwork/lnd/keychain"
 	"github.com/stretchr/testify/require"
+	"golang.org/x/exp/rand"
```

Comment:
Wrong import? I think we instead want to use the `test.RandIntn()` method here for stable/deterministic test vectors.

Reply:
But even then, the test vectors will need to be updated _once_ but will become stable after that.